.buildkite/pipeline.ml.yml

```diff
@@ -411,23 +411,6 @@
   commands:
     - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/build/upload_build_info.sh; fi }; trap cleanup EXIT
     - DATA_PROCESSING_TESTING=1 ARROW_VERSION=nightly ./ci/env/install-dependencies.sh
-    # TODO(scottjlee): Move datasets to train/data-test-requirements.txt 
-    # (see https://github.com/ray-project/ray/pull/38432/)
-    - pip install "datasets==2.14.0"
```

Comment:
just to confirm, is the manual installation for `datasets==2.14.0` no long needed because it's handled gracefully by the new CI workflow?